stories/Button/Button.stories.js

```diff
@@ -86,29 +86,17 @@ const DimensioniComponent = () => (
 
 const ButtonIconComponent = () => (
   <div>
-    <Button color='success' size='lg' className='btn-icon'>
-      <svg className='icon icon-white'>
-        <use xlinkHref='/svg/sprite.svg#it-star-full' />
-      </svg>
-            Icon Button Lg
+    <Button color='primary' size='lg' className='btn-icon'>
+        <Icon color={'icon-white'} /> Icon Button Large
     </Button>{' '}
-    <Button color='primary' className='btn-icon'>
-      <svg className='icon icon-white'>
-        <use xlinkHref='/svg/sprite.svg#it-star-full' />
-      </svg>
-            Icon Button
+    <Button color='success' className='btn-icon'>
+        <Icon color={'icon-white'} /> Icon Button
     </Button>{' '}
     <Button color='danger' size='sm' className='btn-icon'>
-      <svg className='icon icon-secondary'>
-        <use xlinkHref='/svg/sprite.svg#it-star-full' />
-      </svg>
-            Icon Button Sm
+        <Icon color={'icon-primary'} /> Icon Button Small
```

Comment:
As explained above, new Icon component to add `<svg>`s.

Reply:
Got it 👍